raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Certificate of Analysis
+(97) [1] A laboratory designated as an analytical or reference laboratory under this Law shall issue a certificate of analysis on a substance submitted to it pursuant to this Law; [2] The certificate of analysis shall state the methods of analysis followed and be signed by the analyst or reference analyst; [3] A certificate issued pursuant to subsection [1] shall be sufficient evidence of the facts stated in the certification commits an offence and is liable on conviction to a fine not exceeding 5,000 US DOLLARS or imprisonment for a period not exceeding one year or to both.
+
+PART X - INTERNATIONAL OBLIGATIONS 
+
+Harmonization of Regional Environmental Agreements
+(98) [1] The Agency shall advise the State on the harmonization of regional environmental agreements; [2] The Agency shall enter into consultation with other State Agencies in the region and develop Action Plans for the co-operation and harmonization of the management of shared natural resources.
+
+International Environmental Conventions and Treaties on the Environment
+(99) [1] Where Liberia is a party to an international or regional convention or agreement, concerning the management of the environment or natural resources, the Agency shall, in consultation with the relevant Line Ministry: [a] initiate and prepare legislative proposals for consideration by the relevant Ministry for purposes of implementing those international or regional conventions or agreements; [b] identify appropriate measure necessary for the implementation of conventions or agreements; [c] propose the amendment of any enactment other than the Constitution for the purpose of giving effect to the Convention or Treaty; [2] This section applies to any Convention or Treaty whether adopted before or after the coming into force of this Law and whether Liberia became a Party to it after the coming into force of this Law; [3] The Agency, as the national clearinghouse of all environment-related regional and international conventions and agreements, shall coordinate activities related to these instruments in Line Ministries, State agencies and non-governmental organizations, and shall establish procedures for collecting implementation reports from these focal points; [4] The Agency shall make recommendations regarding the ratification of regional and international environmental conventions, treatise and protocols, which it deems necessary for the protection of the environment and natural resources of Liberia; [5] The Agency shall keep a register of all-international conventions or agreements concerning the management of the environment or natural resources to which Liberia is a party.
```

Comment:
Please replace each semi-colon preceding a number in square brackets with a period. Also, please replace:

* `identify appropriate measure` with `identify appropriate measures`
* `treatise and protocols` with `treaties and protocols`
* `all-international conventions` with `all international conventions`